packages/viewer/src/components/viewer-markup/viewer-markup-components.tsx

```diff
@@ -0,0 +1,236 @@
+// eslint-disable-next-line @typescript-eslint/no-unused-vars
+import { FunctionalComponent, h } from '@stencil/core';
+import { Point, Angle, Rectangle } from '@vertexvis/geometry';
+import { DeviceSize } from '../../lib/device';
+import { getBoundingBox2dAnchorPosition } from './utils';
+
+export interface SvgShadowProps {
+  id: string;
+}
+
+export const SvgShadow: FunctionalComponent<SvgShadowProps> = ({ id }) => {
+  return (
+    <filter id={id} filterUnits="userSpaceOnUse">
+      <feGaussianBlur in="SourceAlpha" stdDeviation="2" />
+      <feOffset dx="0" dy="1" result="offsetblur" />
+      <feFlood flood-color="#000000" flood-opacity="0.25" />
+      <feComposite in2="offsetblur" operator="in" />
+      <feMerge>
+        <feMergeNode />
+        <feMergeNode in="SourceGraphic" />
+      </feMerge>
+    </filter>
+  );
+};
+
+export interface BoundingBoxAnchorProps {
+  id?: string;
+  center: Point.Point;
+  angle?: number;
+  width?: number;
+  height?: number;
+  onGrab?: (event: PointerEvent) => void;
+}
+
+export const BoundingBoxAnchor: FunctionalComponent<BoundingBoxAnchorProps> = ({
+  id,
+  angle,
+  center,
+  width,
+  height,
+  onGrab,
+}) => {
+  return (
+    <rect
+      id={id}
+      class="bounds-rect"
+      height={height ?? 8}
+      width={width ?? 8}
+      x={center.x - (width != null ? width / 2 : 4)}
+      y={center.y - (height != null ? height / 2 : 4)}
+      transform={angle ? `rotate(${angle},${center.x},${center.y})` : undefined}
+      onPointerDown={onGrab}
+    />
+  );
+};
+
+export interface BoundingBox1dProps {
+  start: Point.Point;
+  end: Point.Point;
+  deviceSize?: DeviceSize;
+  onStartAnchorPointerDown?: (event: PointerEvent) => void;
+  onCenterAnchorPointerDown?: (event: PointerEvent) => void;
+  onEndAnchorPointerDown?: (event: PointerEvent) => void;
+}
+
+export const BoundingBox1d: FunctionalComponent<BoundingBox1dProps> = ({
+  start,
+  end,
+  deviceSize,
+  onStartAnchorPointerDown,
+  onCenterAnchorPointerDown,
+  onEndAnchorPointerDown,
+}) => {
+  const angle = Angle.normalize(
+    Angle.toDegrees(Angle.fromPoints(start, end)) - 270
+  );
+  const center = Point.create((start.x + end.x) / 2, (start.y + end.y) / 2);
+  const anchorWidth = deviceSize === 'small' ? 14 : 8;
+  const anchorHeight = deviceSize === 'small' ? 14 : 8;
+
+  return (
+    <g>
+      <defs>
+        <SvgShadow id="bounding-box-1d-shadow" />
+      </defs>
+      <g filter="url(#bounding-box-1d-shadow)">
+        <line
+          id="bounding-box-1d-line"
+          class="bounds-line"
+          x1={start.x}
+          y1={start.y}
+          x2={end.x}
+          y2={end.y}
+        />
+        <BoundingBoxAnchor
+          id="bounding-box-1d-start-anchor"
+          center={start}
+          angle={angle}
+          width={anchorWidth}
+          height={anchorHeight}
+          onGrab={onStartAnchorPointerDown}
+        />
+        <BoundingBoxAnchor
```

Comment:
Can we make the selection bounding box and anchors be HTML elements vs SVG? SVG is difficult to customize using CSS variables. I'm thinking it'd be nice for folks to specify circular vs square handles, which can be done if we exposed a variable for border radius.

Reply:
Yeah, I'll take a look at swapping over to that approach. I had run into some challenges with the SVG anchors as I tested with and used these changes, but hadn't done too much investigation into the HTML anchor approach. I'll take another look at that